Makefile

```diff
@@ -10,15 +10,19 @@
 # only.
 #
 # By setting the variable FLOWDB_SERVICE you can choose which flowdb
-# version you'd like to use when running `make up`. Examples:
+# version or versions you'd like to use when running `make up`. Examples:
 #
-#     FLOWDB=flowdb-testdata make up
-#     FLOWDB=flowdb-synthetic-data make up
+#     FLOWDB_SERVICE=flowdb_testdata make up
+#     FLOWDB_SERVICE=flowdb_synthetic_data make up
+#     FLOWDB_SERVICE="flowdb_testdata flowdb_synthetic_data" make up
 #
+# flowmachine and flowapi will connected to the first flowdb service in the list.
 
 DOCKER_COMPOSE_FILE_DEV ?= docker-compose-dev.yml
-FLOWDB_SERVICES ?= flowdb-testdata
+export FLOWDB_SERVICES ?= flowdb_testdata
```

Comment:
Why is the `export` needed here?

(Tbh I didn't even know make supported this - do you know if this a POSIX thing or a GNU Make specific syntax?)